java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -108,6 +108,7 @@ public void visitMethod(MethodTree tree) {
     super.visitMethod(tree);
     BlockTree body = tree.block();
     if (body != null) {
+      DEBUG_MODE_ACTIVATED &= tree.simpleName().name().equals("normalize");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "normalize" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1132)